hashtagsv2/hashtags/views.py

```diff
@@ -53,12 +53,16 @@ def get_queryset(self):
         form = self.form_class(self.request.GET)
         if form.is_valid():
             form_data = form.cleaned_data
-
-            hashtag_qs = hashtag_queryset(form_data)
-
-            if hashtag_qs.count() == 0:
+            if(form_data['project'] == 'www.wikidata.org'):
```

Comment:
I think we could broaden this, in case users attempt to search `www.wikidata.org` or `wikidata.org`. Perhaps we could do `if 'wikidata.org' in form_data['project']`?